src/modules/partition/core/DeviceList.cpp

```diff
@@ -152,6 +152,11 @@ QList< Device* > getDevices( DeviceType which, qint64 minimumSize )
             cDebug() << "  .. Removing too-small" << it;
             it = erase(devices, it );
         }
+        else if ( (*it)->type() == Device::LVM_Device )
```

Comment:
I think the opposite logic would be better. Allow installing only if it is Device::Type::Disk_Device.
Otherwise you'll have to add similar check for other Device types if we introduce them later (maybe RAID, not sure yet if we'll need it).

(Also I would suggest including enum scope i.e. Type, so that at some point we can switch to C++11 scoped enums)